packages/block-library/src/navigation-menu/edit.js

```diff
@@ -79,6 +95,7 @@ function NavigationMenu( {
 		'wp-block-navigation-menu', {
 			'has-text-color': textColor.color,
 			'has-background-color': backgroundColor.color,
+			'has-scroll-x': hasScrollX,
```

Comment:
It seems that the `has-scroll-x'` is not being used in any place of the CSS. Should we remove entirely the js code which handles it?

Reply:
We should if we decide this wrapping approach is valid.